src/core/trulens/core/database/orm.py

```diff
@@ -401,6 +404,86 @@ def parse(
                     dataset_json=obj.model_dump_json(redact_keys=redact_keys),
                 )
 
+        class Event(base):
```

Comment:
I didn't compare this to the event table column defn, but it seems fine otherwise.